### ISSUE TITLE
Missing config options in latest dev version

### ISSUE THREAD

**Issue by bram96** [Alexwijn/SAT#14]
I've just updated from 2.3.2 to the latest develop version and realized that I cannot update my climates anymore. I also found out that the service to set minimum_setpoint is gone and I haven't found a way to change this in the configuration. Was this a deliberate change?

**Comment by Alexwijn** [comment]
The latest version has a redesigned config and setup flow. It's recommended to re-add the integration after removing the current one.

The new options that are available now are only there to tweak the integration. Area's are a fundamental part of the system and should only be configured on setup.

**Comment by bram96** [comment]
I see your point with climates. However I do think that minimum_setpoint is something to tweak. I have removed and added SAT again.

I was using manual PID values and I do know my minimum_setpoint. In the config flow I can choose to either calculate the minimum_setpoint or set it manually or set the PID values. If I set PID values manually I cannot change them afterwards, if I set the minimum_setpoint I cannot change my PID values.
